### ISSUE TITLE
Any string value ending with a digit gets 'px' appended

### ISSUE THREAD

**Issue by nbrombal** [brombal/stylix#1]
It's really nice that Stylix defaults int prop values to use `px` units, but it seems to be a bit overzealous. If I happen to end any Stylix prop value with a digit, it adds the "px" on the end unexpectedly.

For example:
```jsx
<$.div gridArea="slot-1">Some contents</$.div>
```

Renders this style block:
```css
._n23me9 {
    grid-area: place-1px;
}
```

Slapping pretty much any non-numeric character on the end of my string (e.g. "slot-1-") is a functional workaround for the time being.

**Comment by brombal** [comment]
Thanks for the feedback @nbrombal! This was due to a bug in the default-unit PostCSS plugin that Stylix uses. This has been addressed in PR #2. Try it out and let me know if it fixes the issue.

**Comment by nbrombal** [comment]
Finally got a chance to test this out and can confirm that #2 resolves the issue.
